src/components/utils/running-status.ts

```diff
@@ -16,7 +16,7 @@ export function getLoadFlowRunningStatus(
     loadFlowStatus: string
 ): RunningStatus {
     switch (loadFlowStatus) {
-        case 'CONVERGED':
+        case 'COMPLETED':
```

Comment:
I think we want to keep CONVERGED and DIVERGED. It's an important notion for the loadflow

Reply:
done